src/components/Ui/label/index.tsx

```diff
@@ -0,0 +1,7 @@
+export function Label({ title }: { title: string }) {
+  return (
+    <label className="font-canada text-base font-medium capitalize text-[#292929]">
```

Comment:
Quando você fizer o extends, podemos melhorar duas coisas

1. Componente pode ser self-closing (faz o fechamento automático), quando fizer o spread das props o children já vem automatico
2. Fazer spread, para herdar todos os atributos e ajudar na acessibilidade

```suggestion
    <label {...props} className={cx("font-canada text-base font-medium capitalize text-[#292929]", className) />
```